bundles/org.eclipse.equinox.preferences/src/org/eclipse/core/internal/preferences/PreferencesService.java

```diff
@@ -901,7 +902,9 @@ public void setDefaultLookupOrder(String qualifier, String key, String[] order)
 		if (order == null) {
 			DEFAULTS_REGISTRY.remove(registryKey);
 		} else {
-			Arrays.asList(order).forEach(Objects::requireNonNull);
+			if (Stream.of(order).anyMatch(Objects::isNull)) {
```

Comment:
What do you think about using `Arrays.stream()` directly which is called by `Stream.of()`?
At least for me it makes it more clear, that it is a stream of elements of an array and not a stream with a single element order (although this would not be possible with the var-args argument).
```suggestion
			if (Arrays.stream(order).anyMatch(Objects::isNull)) {
```

Reply:
Sure, totally makes sense. I was not aware of that method in `Arrays` but always used the `Stream` operation. Thanks for the hint!

Reply:
Great. Thanks for the update!